src/core/misc.c

```diff
@@ -181,6 +181,24 @@ int strarray_find(char **array, const char *item)
 	return -1;
 }
 
+
+int strarray_find_glob(char **array, const char *item)
+{
+	char **tmp;
+	int index;
+
+	g_return_val_if_fail(array != NULL, 0);
```

Comment:
Hrm, to be honest it might be better to return `-1` instead of 0 to follow the `return index of `item' in `array' or -1 if not found` principle.
(This also applies to `strarray_find`)